ipympl/backend_nbagg.py

```diff
@@ -83,16 +83,8 @@ def __init__(self, canvas, *args, **kwargs):
     def export(self):
         buf = io.BytesIO()
         self.canvas.figure.savefig(buf, format='png', dpi='figure')
-        # Figure width in pixels
-        pwidth = (self.canvas.figure.get_figwidth() *
-                  self.canvas.figure.get_dpi())
-        # Scale size to match widget on HiDPI monitors.
-        if hasattr(self.canvas, 'device_pixel_ratio'):  # Matplotlib 3.5+
-            width = pwidth / self.canvas.device_pixel_ratio
-        else:
-            width = pwidth / self.canvas._dpi_ratio
-        data = "<img src='data:image/png;base64,{0}' width={1}/>"
-        data = data.format(b64encode(buf.getvalue()).decode('utf-8'), width)
```

Comment:
@martinRenou why is it ok to remove this dpi scaling? It seems like we will now have imperfect behavior on hi dpi displays.

Reply:
This might have been a mistake, we should open an issue to track this

Reply:
I'll do it, actually.  If I understand correctly, the PNG doesn't get saved at high res.

Reply:
Note: we should not determine the dpi based on the browser because it could differ between clients.
It should be a fixed value.